src/lib/pages/frontsite/Literature/LiteratureSearch/SearchMessage/SearchMessage.js

```diff
@@ -16,24 +18,53 @@ export default class SearchMessage extends Component {
   };
 
   render() {
+    const { currentResultsState } = this.props;
+    const totalResults = currentResultsState.data.total;
+
     const requestFormLink = (
-      <Link className="primary" to={this.onClickBookRequestLink()}>
-        request form
+      <Link
+        className={totalResults != 0 ? 'primary' : 'dark'}
+        to={this.onClickBookRequestLink()}
+      >
+        this form
       </Link>
     );
-    return (
-      <Message icon info>
-        <Icon name="info circle" />
-        <Overridable id="SearchMessage.content">
-          <Message.Content>
-            <Message.Header>
-              Couldn't find the literature you were looking for?
-            </Message.Header>
-            Please fill in the {requestFormLink} to request new literature from
-            the library. (Login required)
-          </Message.Content>
-        </Overridable>
-      </Message>
-    );
+
+    if (totalResults != 0) {
+      return (
+        <Message icon info>
+          <Icon name="info circle" />
+          <Overridable id="SearchMessage.content">
+            <Message.Content>
+              <Message.Header>
```

Comment:
Shall we simply add/remove the `Header` based on the number of results, and keep the rest the same?

```
const Header = totalResults > 0 ? (
   <Message.Header>
  Couldn't find the literature you were looking for?
              </Message.Header>
) : null;
```